editor/tests/org.fusesource.ide.wsdl2rest.ui.tests.integration/src/main/java/org/fusesource/ide/wsdl2rest/ui/tests/integration/Wsdl2RestWizardIT.java

```diff
@@ -0,0 +1,112 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/ 
+package org.fusesource.ide.wsdl2rest.ui.tests.integration;
+
+import java.io.File;
+import java.net.URL;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import org.eclipse.core.resources.IContainer;
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.core.resources.IResourceVisitor;
+import org.eclipse.core.runtime.CoreException;
+import org.fusesource.ide.camel.model.service.core.tests.integration.core.io.FuseProject;
+import org.fusesource.ide.wsdl2rest.ui.wizard.Wsdl2RestWizard;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+
+/**
+ * Test to ensure we can run the wsdl2rest wizard.
+ * 
+ * @author brianf
+ */
+public class Wsdl2RestWizardIT {
+	
+    static final String WSDL_LOCATION = "src/test/resources/wsdl/Address.wsdl"; //$NON-NLS-1$
+    static final String OUTPUT_PATH = "target/generated-wsdl2rest"; //$NON-NLS-1$
+    static final String SPRING_CAMEL_PATH = "/src/main/resources/META-INF/spring/doclit-camel-context.xml"; //$NON-NLS-1$
+
+    @Rule
+	public FuseProject fuseProject = new FuseProject(Wsdl2RestWizardIT.class.getName());
+
+	@Test
+	public void testWsdl2RestWizardFinish() throws Exception {
+    	Wsdl2RestWizard wizard = new Wsdl2RestWizard();
+        File wsdlFile = new File(WSDL_LOCATION);
+        Path outpath = new File(OUTPUT_PATH).toPath();
```

Comment:
indentation issue, please merge first https://github.com/bfitzpat/fuseide/pull/24 to avoid potential conflict